.eslintrc.json

```diff
@@ -0,0 +1,34 @@
+{
+    "env": {
+        "browser": true,
+        "es6": true,
+        "node": true
+    },
+    "extends": [
+        "plugin:react/recommended",
+        "airbnb"
+    ],
+    "globals": {
+        "Atomics": "readonly",
+        "SharedArrayBuffer": "readonly"
+    },
+    "parserOptions": {
+        "ecmaFeatures": {
+            "jsx": true,
+            "modules": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
+        "no-shadow": "off",
+        "no-param-reassign": "off",
+        "eol-last": "off",
+        "arrow-parens": "off",
+        "import/no-unresolved": "off",
+        "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
+      }
+}
```

Comment:
This configuration is not the one provided by Microverse, please refer to the [React and Redux Linter Repo](https://github.com/microverseinc/linters-config/tree/master/react-redux) and get the config files from there.